Week2/ex3/joins.js

```diff
@@ -0,0 +1,21 @@
+const mysql = require("mysql");
+
+const connection = mysql.createConnection({
+  host: "localhost",
+  user: "hyfuser",
+  password: "hyfpassword",
+  database: "company",
+});
+
+connection.connect((err) => {
+  if (err) throw err;
+  console.log("Connected to DB..");
+});
+
+connection.query("SELECT full_name, manager_no FROM employee; ");
+
+connection.query(
+  "SELECT employee.full_name AS name, department.title AS department FROM employee JOIN department ON employee.dept_no=department.dept_no;"
```

Comment:
The exercise is:
> Write a query that retrieves all employees and their working department title. If no employee worked in a specific department, return the department too.

But with this query excludes departments that don't have any employees.
Hint: LEFT/RIGHT JOIN.